src/test/java/io/deepstream/ErrorCallback.java

```diff
@@ -0,0 +1,8 @@
+package io.deepstream;
+
+import io.deepstream.constants.Event;
+import io.deepstream.constants.Topic;
+
+public interface ErrorCallback {
```

Comment:
This is a little cheeky, but given we want to expose the ErrorCallback in all our tests it does make mocking easier. Not sure how bad of a bad code practice this is